data-raw/calsim_2019_BiOp_ITP/clean_calsim.R

```diff
@@ -0,0 +1,122 @@
+library(tidyverse)
+library(lubridate)
+library(stringr)
+library(devtools)
+
+cvpia_nodes <- read_csv('data-raw/biop_inputs/cvpia_calsim_nodes.csv', skip = 1)
+need_split <- cvpia_nodes$calsim_habitat_flow %>% str_detect(', ')
+habitat_split <- cvpia_nodes$calsim_habitat_flow[need_split] %>% str_split(', ') %>% flatten_chr()
+habitat_nodes <- c(cvpia_nodes$calsim_habitat_flow[!need_split], habitat_split, 'C134', 'C160')[-20]
+
+need_split <- cvpia_nodes$cal_sim_flow_nodes %>% str_detect(', ')
+div_split <- cvpia_nodes$cal_sim_flow_nodes[need_split] %>% str_split(', ') %>% flatten_chr()
```

Comment:
consider renaming these `need_split` variables to be more specific to their use in the script

Reply:
updated